src/frontend/OverlayManager/Overlay/NavBarOverlayWrapper/index.tsx

```diff
@@ -15,12 +15,22 @@ import { useFlags } from 'launchdarkly-react-client-sdk'
 
 export function NavBarOverlayWrapper({ appName, runner }: BrowserGameProps) {
   const location = useLocation()
-  const { pathname } = location
+  const { pathname, search } = location
   const [collapsed, setCollapsed] = useState(false)
   const { t } = useTranslation()
   const comingSoonText = t('overlay.comingSoon', 'Coming Soon')
   const flags = useFlags()
 
+  const searchParams = new URLSearchParams(search)
+  const urlQueryParam = searchParams.get('url')
+
+  useEffect(() => {
+    window.api.trackScreen('Overlay Page', {
+      pathname: pathname === '/' ? '/quests' : pathname,
```

Comment:
without this, the metrics event will be confusing and if we change the '/' route, we want the same events to still be sent for each screen